### ISSUE TITLE
Is it possible to evaluate the model?

### ISSUE THREAD

**Issue by catlisboa** [trevorstephens/gplearn#120]
Hi! First of all, thank you very much for your implementation! 

My question:
Is there anyway to evaluate the model (made with a train set), with a test set?

**Comment by DaGuT** [comment]
```
   from sklearn.metrics import classification_report, confusion_matrix

  ....tons of your code here xD

    pred = gp.predict(X_test)
    print('Confusion Matrix')
    print(confusion_matrix(Y_test, pred))
    print('Classification Report')
    print(classification_report(Y_test, pred))
```


________________

Result is something like:

> Confusion Matrix
> [[18  2]
>  [10  9]]
> Classification Report
>              precision    recall  f1-score   support
> 
>         0.0       0.64      0.90      0.75        20
>         1.0       0.82      0.47      0.60        19
> 
> avg / total       0.73      0.69      0.68        39

**Comment by trevorstephens** [comment]
@catlisboa you can use any sklearn cross validation method to evaluate your models. https://scikit-learn.org/stable/modules/cross_validation.html

**Comment by catlisboa** [comment]
Thank you very much!
